pages/firststeps.md

```diff
@@ -45,6 +45,10 @@ Review these [Vagrant instructions](vagrant.md) to ensure that you have fully co
 
 Follow the instructions on the [GitHub and Markdown page](githubandmarkdown.md). Make sure that you've linked to your github.io and your personal .md page on the [Gitter chat](https://gitter.im/open-learning-exchange/chat) (&lt;username&gt;.github.io and &lt;username&gt;.github.io/#!pages/profiles/&lt;username&gt;.md).
 
+Here is the tutorial as reference to work on the Step 3 and Step 5.
+Markdown Tutorial:
```

Comment:
I edited what you told me to do. Could you really approve the pull request please?